apollo-router/src/query_planner/selection.rs

```diff
@@ -166,19 +241,23 @@ mod tests {
                 values.push(value);
             });
 
-        Ok(Value::Array(
+        let res = Ok(Value::Array(
             values
                 .into_iter()
-                .flat_map(|value| match (value, selections) {
+                .map(|value| {
+                    execute_selection_set(value, selections, schema, None)
+                    /*match (value, selections) {
                     (Value::Object(content), requires) => {
-                        select_object(content, requires, schema).transpose()
+                        select_object(content, requires, schema) //.transpose()
                     }
-                    (_, _) => Some(Err(FetchError::ExecutionInvalidContent {
+                    (_, _) => Err(FetchError::ExecutionInvalidContent {
                         reason: "not an object".to_string(),
-                    })),
+                    }),*/
```

Comment:
uh did we remove a test? :D

Reply:
no, the `select_object` was just replaced with `execute_selection_set`